CONTRIBUTING.md

```diff
@@ -45,6 +45,7 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
     - For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k.
         - Note that for brands that have already been added the threshold for continued inclusion rises to 750k.
     - For GitHub projects, the amount of "stars" should be above 5k.
+    - For a well known or widely accepted de facto standard logo (where official sources don't exist), popularity will be assesed on a case-by-case basis by maintainers. For example, Javascript falls in this category.
```

Comment:
> For a well known or widely accepted de facto standard logo (where official sources don't exist)

I think we should pick one from "well known," "widely accepted," "de facto," and "standard," instead of all four, to make the language simpler.

Reply:
make the choice @sachinraja. I don't really prefer any over the others

Reply:
"widely accepted" makes the most sense to me.

Reply:
> "widely accepted" makes the most sense to me

To me too.

Reply:
widely accepted it is, then.